cg/constants/constants.py

```diff
@@ -43,21 +43,25 @@ def actions(cls) -> list[str]:
         return list(cls)
 
 
-COLLABORATORS = ("cust000", "cust002", "cust003", "cust004", "cust042")
-
-COMBOS = {
-    "DSD": ("DSD", "DSD-S", "HYP", "SEXDIF", "SEXDET"),
-    "CM": ("CNM", "CM"),
-    "Horsel": ("Horsel", "141217", "141201"),
-}
-
 CONTAINER_OPTIONS = ("Tube", "96 well plate", "No container")
 
 CONTROL_OPTIONS = ("", "negative", "positive")
 
 DEFAULT_CAPTURE_KIT = "twistexomerefseq_9.1_hg19_design.bed"
 
 
+class CustomerId(StrEnum):
+    CG_INTERNAL_CUSTOMER: str = "cust000"
+    cust001: str = "cust001"
```

Comment:
```suggestion
    CUST001: str = "cust001"
```
similar for the others

Reply:
Fixed